src/services/ProjectTokenHistoricalPriceService.ts

```diff
@@ -46,8 +47,10 @@ export class ProjectTokenHistoricalPriceService {
     }
 
     const startingSeconds = lbpMgr.startTime.getTime() / 1000;
-    const intervalMinutes = 60/*min*/;
-    const intervalSeconds = intervalMinutes * 60/*sec*/;
+    const timeWentLiveSeconds = (await lbpMgr.lbp.contract.getGradualWeightUpdateParams())[0].toNumber();
```

Comment:
Please get start and end times and weights from our entities.  they are all there.  Or if not, we should add them.

same thing for any contract interactions.  please keep them wrapped in the appropriate entities.